website/docs/install-config/configuration/configuration.mdx

```diff
@@ -192,29 +192,123 @@ When your PostgreSQL databases are running behind a connection pooler (like PgBo
 
 - `AUTHENTIK_CHANNEL__URL`: Channel layers configuration URL, uses [the Redis Settings](#redis-settings) by default
 
-## Broker Settings
+## Worker settings
 
-- `AUTHENTIK_BROKER__URL`: Broker configuration URL, defaults to Redis using [the respective settings](#redis-settings)
-- `AUTHENTIK_BROKER__TRANSPORT_OPTIONS`: Base64-encoded broker transport options
+##### `AUTHENTIK_WORKER__PROCESSES`
 
-    :::info
-    `AUTHENTIK_REDIS__CACHE_TIMEOUT_REPUTATION` only applies to the cache expiry, see [`AUTHENTIK_REPUTATION__EXPIRY`](#authentik_reputation__expiry) to control how long reputation is persisted for.
-    :::
+Configure how many Dramatiq worker processes should be started.
+
+Defaults to 2. A value below 2 workers is not recommended. In environments where scaling with multiple replicas of the authentik server is not possible, this number can be increased to handle higher loads.
```

Comment:
Yeah, I guess I am worried that people might have in mind the Background worker, which I assume is diff from this Dramatiq worker...